hicommonwealth/commonwealth
# Fix thread body edit + comment text edit

CRUD for threads and comments should now work correctly.

## Link to Issue
Closes: #3126 
Closes: #3128 
Closes: #3001 

## Description of Changes
- Fixes issue where the thread body could not be edited
- Fixes issue where a thread comment could not be edited
- Fixes issue where deleting a comment did not refresh the UI
- Fixes issue where comment cancel button does nothing

## Test Plan
- Create a thread, attempt to update the title and body, confirm that it saves and the page is refreshed
- Edit the thread, but click cancel– confirm that the confirmation modal behaves correctly
- Create a comment on a thread, attempt to edit the comment text, confirm that it saves and the page is refreshed
- Edit a comment, but click cancel– confirm that the confirmation modal behaves correctly
- Delete a comment– confirm that the UI is refreshed

## Deployment Plan
N/A

## Other Considerations
N/A
